src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.CreatingReport;
+import core.basesyntax.service.FileHandler;
+import core.basesyntax.service.GenerationDataForStorage;
+import core.basesyntax.service.ProcessingDataFromFile;
+import core.basesyntax.service.impl.CreatingReportImpl;
+import core.basesyntax.service.impl.FileHandlerImpl;
+import core.basesyntax.service.impl.GenerationDataForStorageImpl;
+import core.basesyntax.service.impl.OperationStrategyImpl;
+import core.basesyntax.service.impl.ProcessingDataFromFileImpl;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.impl.Balance;
+import core.basesyntax.strategy.impl.Purchase;
+import core.basesyntax.strategy.impl.ReturnFruit;
+import core.basesyntax.strategy.impl.Supply;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    private static final String OPERATIONS_PATH = "src/main/java/operations.csv";
+    private static final String REPORT_PATH = "src/main/java/report.csv";
+
+    public static void main(String[] args) {
+        final Map<FruitTransaction.Operation, OperationHandler> operationOperationHandlerMap
+                = new HashMap<>();
+        operationOperationHandlerMap.put(FruitTransaction.Operation.BALANCE, new Balance());
+        operationOperationHandlerMap.put(FruitTransaction.Operation.PURCHASE, new Purchase());
+        operationOperationHandlerMap.put(FruitTransaction.Operation.RETURN, new ReturnFruit());
+        operationOperationHandlerMap.put(FruitTransaction.Operation.SUPPLY, new Supply());
+
+        FileHandler fileHandler = new FileHandlerImpl();
+        ProcessingDataFromFile processData = new ProcessingDataFromFileImpl();
```

Comment:
I think it's bad name ProcessingDataFromFile because in this place you don't work with file